puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/test/testForkliftCentosDevel.groovy

```diff
@@ -0,0 +1,40 @@
+pipeline {
+    agent none
+
+    options {
+        timestamps()
+            timeout(time: 2, unit: 'HOURS')
+            disableConcurrentBuilds()
+            ansiColor('xterm')
+    }
+
```

Comment:
Even if you only have one stage, I think you still need stages {} EG: https://github.com/theforeman/foreman-infra/blob/master/puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/3.9.groovy#L11